vendor/modules.txt

```diff
@@ -37,25 +47,33 @@ github.com/ClickHouse/clickhouse-go/lib/protocol
 github.com/ClickHouse/clickhouse-go/lib/types
 github.com/ClickHouse/clickhouse-go/lib/writebuffer
 # github.com/DATA-DOG/go-sqlmock v1.5.0
+## explicit
 github.com/DATA-DOG/go-sqlmock
 # github.com/DataDog/zstd v1.4.1
 github.com/DataDog/zstd
 # github.com/EagleChen/mapmutex v0.0.0-20180418073615-e1a5ae258d8d
+## explicit
 github.com/EagleChen/mapmutex
 # github.com/EagleChen/restrictor v0.0.0-20180420073700-9b81bbf8df1d
+## explicit
```

Comment:
@lvrach What are these `explicit` lines about?

Reply:
> As of Go 1.14, the go command will also verify that your project’s vendor/modules.txt file is consistent with its go.mod file. If not, you may get an error, such as the one shown below, when building your project:

So for every entry in the the go.mod you have a `## explicit` comment annotation.